easybuild/easyconfigs/s/SDSL/deactivate-failed-test.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru sdsl-lite-c32874cb2d8524119f25f3b501526fe692df29f4.orig/test/cst_byte_test.cpp sdsl-lite-c32874cb2d8524119f25f3b501526fe692df29f4/test/cst_byte_test.cpp
```

Comment:
@mboisson please add author and short description to this patch?
also, please rename this file to SDSL-<version>_deactivate-failed-test.patch?

same for the other patch